### DISCUSSION TITLE
Location of "Partition coefficient (interstitial/water)"

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#1440] [Questions & Problems] [answered]
I am curious as to where the parameter "Partition coefficient (interstitial/water)" is defined. I can see it in the simulation, but I do not see it in the spatial structure. How and where is it defined?

![image](https://user-images.githubusercontent.com/57540473/213155240-962c5c36-0405-4d13-863d-0b0ee9ee0821.png)
![image](https://user-images.githubusercontent.com/57540473/213155316-532b60a9-3353-424d-a0ee-f2f1532d00a2.png)

**Comment by PavelBal** [comment]
This a "dynamic" parameter created during simulation creation. I think it is defined somewhere in the file `AllCalculationMethods.pkml` located in the MoBi installation folder.

**Comment by Yuri05** [comment, marked as answer]
as Pavel said :)
it is defined in "C:\ProgramData\Open Systems Pharmacology\MoBi\11.0\AllCalculationMethods.pkml"

The rationale behind it was: different calculation methods for partition coefficients, cellular permeabilities, etc. require different inetrmediate parameters for their calculation. To avoid "overloading" the spatial structure with all those parameters (which would be also completely irrelevant if a particular calculation method is not used by any molecule involved) this "dynamical" parameter creation procedure was chosen.

And like everything, it has advantages and disadvantages :)
https://github.com/Open-Systems-Pharmacology/MoBi/issues/502
